src/marks/frame.d.ts

```diff
@@ -2,11 +2,30 @@ import type {InsetOptions} from "../inset.js";
 import type {MarkOptions, RenderableMark} from "../mark.js";
 
 export interface FrameOptions extends MarkOptions, InsetOptions {
+  /**
+   * Only renders that side of the frame. The *rx*, *ry*, *fill*, and
+   * *fillOpacity* options are then ignored.
```

Comment:
Implied here is that the **anchor** defaults to null, and these comments apply when the **anchor** is not null.

Reply:
maybe

```suggestion
   * If not null, only renders that side of the frame. The *rx*, *ry*, *fill*, and
   * *fillOpacity* options are then ignored.
```